CHANGES.md

```diff
@@ -1,3 +1,8 @@
+# UNRELEASED
+- [FIXED] Always retry failed requests made by the IAM plugin.
```

Comment:
I think this could be confusing because `retry` is most often used wrt to the `retry` plugin, how about:
`Stopped disabling the IAM auth plugin after failed IAM authentications. Subsequent requests will re-request authorization, potentially failing again if the original authentication failure was not temporary.`
Bit of a mouthful 😕